gen_err.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import random
+import re
+
+__action_pattern_map = {
+    'delete(': ("\(", ""),
+    'delete)': ("\)", ""),
+    'delete,': (",", ""),
+    'delete:': (":", ""),
+    'delete{': ("\{", ""),
+    'delete}': ("\}", ""),
+    'delete[': ("\[", ""),
+    'delete]': ("\]", ""),
+    'delete+': ("\+", ""),
+    'delete-': ("-", ""),
+    'delete=': ("=", ""),
+    'delete<': ("<", ""),
+    'delete>': ("\*", ""),
+    'duplicate(': ("\(", "( ("),
+    'duplicate)': ("\)", ") )"),
+    'duplicate,': (",", ", ,"),
+    'duplicate{': ("\{", "{ {"),
+    'duplicate}': ("\}", "} }"),
+    'duplicate[': ("\[", "[ ["),
+    'duplicate]': ("\]", "] ]"),
+    'replace,with;': (",", ";"),
+}
+
+
+__action_pattern_map_kw = {
+    'deleteWhile': ("while", ""),
+    'deleteElif': ("elif", ""),
+    'deleteIf': ("if", ""),
+    'deleteIf': ("for", ""),
+    'replaceDefWithDf': ("def", "df"),
+    'replaceElseWithEles': ("else", "eles"),
+    'replaceWhileWithFor': ("while", "for"),
+    'replaceForWithWhile': ("for", "while"),
+    'replaceWhileWithWhil': ("while", "whil"),
+    'replaceClassWithCls': ("class", "cls"),    
+    'replacePandasWithPanda': ("pandas", "panda"),
+    'replace<INDENT>With<DEDENT>': ("<INDENT>", "<DEDENT>"),
+    'replaceNumpyWithNump': ("numpy", "nump"),
+    'replaceRandomWithRandm': ("random", "randm"),
+    'replaceImportWithImport': ("import", "Import"),
+    'replaceIfWithElse': ("if", "else"),
+    'replaceElseWithElif': ("else", "elif"),
+    'replaceElifwithElseif': ("elif", "elseif"),
+}
+
+_actions = list(__action_pattern_map.keys())
+_actions_kw = list(__action_pattern_map_kw.keys())
+
+def corrupt_syntax(line):
+
+    err_cnt = random.randint(0, 2)
+    cnt = 0
+    np.random.shuffle(_actions)
+
+    for act in _actions:
+        if cnt > err_cnt:
+            break
+
```

Comment:
is it a good practice to leave an empty line after if condition.

Can you please make sure we follow style guide for python code